i18n/md/collections/2023/en/categories/nodejs-framework.md

```diff
@@ -0,0 +1,18 @@
+---
+id: nodejs-framework
+language: en
+---
+
+This year too {nextjs} is the leading project in the backend/fullstack category.
+
+The [version 14](https://nextjs.org/blog/next-14) focuses on improving server actions and introduces a new Partial Preview rendering mode.
+
+With React Server Components, developers can build web applications with new paradigms that can simplify drastically the code: fetching data directly from a component that is made `async` and mutating data by binding forms to server actions.
+
+A lot has been said about [a presentation](https://www.youtube.com/watch?v=9CN9RCzznZc&t=0s) that shows a SQL request made directly from a component but people may have missed the point: React Server Components can behave like self-contained components that can be dropped any where in your application, without any tedious wiring, the video does a great job to help building the new mental model made possible by RSC, showing trees of component mixing seamlessly server and client components.
```

Comment:
> A lot has been said about [**this** presentation](https://www.youtube.com/watch?v=9CN9RCzznZc&t=0s) **that showcases making an SQL request directly from a component,** but people may have missed the point: React Server Components can behave like self-contained components that can be dropped any where in your application, without any tedious wiring, the video does a great job to help building the new mental model made possible by RSC, **showing a mix of server and client components that work seamlessly.**


I'm not sure if I lost the spirit of `showing trees of component mixing seamlessly server and client components.` or not here, maybe I misunderstood the message.